src/Client.rei

```diff
@@ -133,7 +133,9 @@ type response('response) =
 type clientResponse('response) = {
   data: option('response),
   error: option(CombinedError.t),
+  extensions: option(Js.Dict.t(string)),
   response: response('response),
+  stale: option(bool),
```

Comment:
My memory sometimes doesn't serve me too well but I believe this field is guaranteed

Reply:
So it _is_ guaranteed on the responses handed to hooks, but not on `OperationResult`, which is what gets returned by Client.* methods. So in this case we pass it through with the `option` wrapper and let folks decode it if they like.